.babelrc

```diff
@@ -1,3 +1,15 @@
 {
-    "presets": ["es2015", "react", "stage-0"]
+    "presets": [
+        "@babel/react",
+        [
+            "@babel/preset-env",
+            {
+                "modules": false
+            }
+        ]
+    ],
+    "plugins": [
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-transform-runtime"
+    ]
 }
```

Comment:
I was wondering why the config was changed to these specific settings. If we want to migrate this app to CRA anyway, maybe their preset would be useful for us for example: https://github.com/facebook/create-react-app/tree/master/packages/babel-preset-react-app